### PR TITLE
Added maintenance mode.

### PR DESCRIPTION
**Pull request by desecho**

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -27,6 +27,8 @@ django_src_path: "{{ django_project_path }}/src"
 django_static_path: "{{ django_path }}/static"
 # Path for MEDIA_ROOT
 django_media_path: "{{ django_path }}/media"
+django_maintenance_path: "{{ django_project_path }}/maintenance"
```
Comment (ellmetha) [inline defaults/main.yml]: Could you add a comment here in order to explicit the purpose of this directory ? :wink: eg. explaining that it is used to store maintenance pages, etc.

**Review by ellmetha — CHANGES_REQUESTED** [review]
I think that you forgot to include the `maintenance.html` template into your branch! :smile_cat:

**Comment by desecho** [comment]
I made the changes

**Comment by ghost** [comment]
I've tested it on our local dummy django projet, in vagrant mode, and this PR breaks the scenario where we have a `django_project_symlink_dest` because the maintenance folder was created and then the symlink can't happen.

This failure highlights that the `maintenance` folder should be in `{{ django_path }}/maintenance` instead of `{{ django_project_path }}/maintenance`.

**Comment by desecho** [comment]
I changed the path to `{{ django_path }}/maintenance`.

**Review by ellmetha — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**